### PR TITLE
Week1 databases homework

### PR DESCRIPTION
**Pull request by ShravyaGurmitkal**

### CODE REVIEW

File: databases/week1/hw_Week1.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+--1. Find out how many tasks are in the task table
+SELECT COUNT(*) AS 'num_of_tasks' FROM `task`;
+
+--2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) AS 'num_of_tasks' FROM task WHERE `due_date` IS NULL;
+
+--3. Find all the tasks that are marked as done
+SELECT * FROM `task` WHERE status_id = (SELECT `id` FROM `status` WHERE `name` = 'Done');
```
Comment (RZhyvitskyi) [inline databases/week1/hw_Week1.sql:8]: Wow, very interesting solution instead of JOIN. Very interesting.

File: databases/week1/hw_Week1.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+SELECT * FROM `task` ORDER BY created DESC;
+
+--6. Get the single most recently created task
+SELECT * FROM `task` ORDER BY created DESC LIMIT 1;
+
+--7. Get the title and due date of all tasks where the title or description contains database
+SELECT `title`, `due_date` FROM `task` WHERE `title` LIKE '%database%' OR `description` LIKE '%database%';
+
+--8. Get the title and status (as text) of all tasks
+SELECT CONCAT(task.title, ' ', status.name) AS `text` FROM `task`
```
Comment (RZhyvitskyi) [inline databases/week1/hw_Week1.sql:23]: Very good with CONCAT

**Review by RZhyvitskyi — APPROVED** [review]
I like your work. I can learn many interesting ways how to make the task from you!

**Comment by ShravyaGurmitkal** [comment]
> Pls, implement given feedbacks!

Hi @Elia-Git ,

I can't find any of your feedback on this PR to implement them.

**Pull request merged**